core/protobuf/src/proto/journal/journal.proto

```diff
@@ -41,16 +44,17 @@ message JournalEntry {
   // deprecated, lineage is no longer supported
   optional LineageIdGeneratorEntry lineage_id_generator = 14;
   optional MergeStoreEntry merge_store = 26;
-  optional NewBlockEntry new_block = 35;
+  optional NewBlockEntry new_block = 38;
```

Comment:
why are these and other entries renumbered?

Reply:
the renumbering is needed, because on the enterprise version we used certain ids for the active sync related entries. This is to make sure we are consistent with that.  Enterprise version is released, and we have yet to release any of the open source versions containing these entries, so I figured changing these numbers are safe to do.